website/source/docs/install/index.html.md

```diff
@@ -9,19 +9,55 @@ description: |-
 
 # Install Consul
 
-Installing Consul is simple. There are two approaches to installing Consul:
+Installing Consul is simple. There are two main approaches to installing Consul:
 
-1. Using a [precompiled binary](#precompiled-binaries)
+1. [Installing on Kubernetes](#install-on-kubernetes) 
 
-1. Installing [from source](#compiling-from-source)
+1. Installing as a standalone binary 
 
-Downloading a precompiled binary is easiest, and we provide downloads over TLS
-along with SHA256 sums to verify the binary. We also distribute a PGP signature
-with the SHA256 sums that can be verified.
+   a. Using a [precompiled binary](#precompiled-binaries)
 
-The [Getting Started guides](https://learn.hashicorp.com/consul/getting-started/install?utm_source=consul.io&utm_medium=docs) provide a quick walkthrough of installing and using Consul on your local machine.
+   b. Installing [from source](#compiling-from-source)
 
-## Precompiled Binaries
+   Downloading a precompiled binary is easiest, and we provide downloads over TLS
+   along with SHA256 sums to verify the binary. We also distribute a PGP signature
+   with the SHA256 sums that can be verified.
+
+The Getting Started guides provide a quick walkthrough of installing and using Consul as a
+
+* [Standalone binary your local machine](https://learn.hashicorp.com/consul/getting-started/install?utm_source=consul.io&utm_medium=docs). 
+* [DaeemonsSet on Minikube on your local machine](https://learn.hashicorp.com/consul/kubernetes/minikube?utm_source=consul.io&utm_medium=docs&utm_content=k8s&utm_term=mk)
+
+## Install on Kubernetes
+
+Installing Consul on Kubernetes is done through Helm, which provides an easy way to deploy applications on Kubernetes. 
+
+Determine the latest version of the Consul Helm chart
+by visiting [https://github.com/hashicorp/consul-helm/releases](https://github.com/hashicorp/consul-helm/releases).
```

Comment:
We don't typically use full urls in docs.

Reply:
This was copied and pasted from the Kubernetes install docs, how would you link to a Github releases page from this doc?

Reply:
We shouldn't be copying this doc. Just link to the existing install guide. Otherwise we have to update things in multiple places which inevitably we'll forget to do.